src/pages/layout/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Layout } from '@/components/Layout'
+
+export default function IndexPage() {
+  return (
+    <Layout>
```

Comment:
IMO we just need Header, Content, Sidebar, ToC and Footer for now. Check the [wireframes](https://excalidraw.com/#json=IR_1swMEP3fId2zgoNBLi,113Iq6VmczCOFHS0k28QAQ)

Reply:
Dnt know where is the complexity. Decouplig the layout structure to what it has placed can fix many different bugs and make it maintained once. and not on every content-section element

Reply:
At the same time, there are many thinks to take care when placing elements, many different responsivity rules that if we do it per element we will have to deal every single time on each. If we declare the Layout slots available, the layout will take care of the responsivity and developer will only have to deal with where to place it (not when it is shown or how much size should it take or where in the layout structure should it be fitted)

Reply:
At the same time, `simple` is a false-friend that means easy-understondable, easy-to-maintain, easy-to-use, (`sencillo`in spanish). For sure, `plain` or `basic` (`simple` in spanish) is one of the possible solutions to make it `simple` but declaring Layout slots for wrapping and placing the elements which manages the responsivity once reducing bugs (breakpoint hide/show, hamb-menu appearance, layout shifting) and giving the opportunity to only have to wrap your component in the right/desired Layout Subelement does't add complexity and its just the opposite.

Reply:
> IMO we just need Header, Content, Sidebar, ToC and Footer for now (check the [wireframes](https://excalidraw.com/#json=IR_1swMEP3fId2zgoNBLi,113Iq6VmczCOFHS0k28QAQ)). I think that is better to keep it simple and if we need them in the future we could add them

This extra simplicity at the beginning could lead to custom layouts and inconsistency later in time when we start including complexity.
I suggest we have the same layout in all pages done and validated from day 1.